tower-batch/tests/ed25519.rs

```diff
@@ -84,24 +86,37 @@ impl Drop for Ed25519Verifier {
 
 // =============== testing code ========
 
-async fn sign_and_verify<V>(mut verifier: V, n: usize) -> Result<(), V::Error>
+async fn sign_and_verify<V>(
```

Comment:
This has me musing over how we could possibly define transcripts like generators